foundations/html_css/html-foundations/html-boilerplate.md

```diff
@@ -121,10 +121,10 @@ The HTML boilerplate in the `index.html` file is complete at this point, but how
 
 1. You can find the HTML file in your file system and then double click it. This will open up the file in the default browser your system uses.
 
-1. You can use the terminal to open the file in your browser.
-
-    - `Ubuntu` - Navigate to the directory containing the file and type `google-chrome index.html`
-    - `macOS` - Navigate to the directory containing the file and type `open ./index.html`
+1. You can use the terminal to open the file in your browser:
+  - Ubuntu: Navigate to the directory containing the file and type `google-chrome index.html`
+  - macOS: Navigate to the directory containing the file and type `open ./index.html`
+  - WSL: Navigate to the directory containing the file and type `explorer.exe index.html`. Note, this will open up the file in the default browser your system uses.
```

Comment:
Awesome! If you fix the nested list indentation to be 3 spaces instead of 2, it'll correctly recognise the UL as a nested list and resolve all of the linting errors.
```suggestion
   - Ubuntu: Navigate to the directory containing the file and type `google-chrome index.html`
   - macOS: Navigate to the directory containing the file and type `open ./index.html`
   - WSL: Navigate to the directory containing the file and type `explorer.exe index.html`. Note, this will open up the file in the default browser your system uses.
```

Reply:
Thanks! I couldn't figure out how many spaces to put in, and the style guide appears to be out of sync with the linter.

Reply:
The style guide calls for a 3-space indent for lists nested within an ordered list.
The linter was giving different things because it misinterpreted the inner list not as a nested list, but a separate top-level list (just built-in rule precedence, not much we cam do about that), so gave the errors in accordance with that.